packages/templates/README.md

```diff
@@ -0,0 +1,9 @@
+# Templates
+
+Page layouts allow people to apply individual pages defined in templates to their story without choosing an entire template. Page layouts are based on the same JSON definitions as templates defined in `src/raw`.
```

Comment:
This describes page layouts, not templates.

Let's use something simple like

```
This package contains the templates used by the dashboard and the editor (as part of page layouts).
```

Reply:
Fixed